docs/multicluster/user-guide.md

```diff
@@ -160,7 +160,7 @@ member clusters.
 * Create below `ClusterClaim` and `ClusterSet` in the member cluster `test-cluster-east`.
 
 Note: Update `server: "https://172.18.0.2:6443"` in `ClusterSet` resource to
-the correct the leader cluster API address.
+the correct the leader cluster API address. You can refer to the template YAML file `./multicluster/config/samples/clusterset_init/multicluster_membercluster_template.yaml`
```

Comment:
```suggestion
the correct the leader cluster API address. You can refer to the template YAML file `[multicluster_membercluster_template.yaml](../../multicluster/config/samples/clusterset_init/multicluster_membercluster_template.yaml)`
```
Perhaps more friendly to use a link for reference.

Reply:
thanks for comments. Done.